### ISSUE TITLE
donation strip on Foyer doesn't work

### ISSUE THREAD

**Issue by helenvarley** [upstage-org/upstage#1300]
i've had feedback that the donation strip on the Foyer doesn't work - the person trying to donate entered the amount and credit card details but nothing happened when she clicked on the donate button to complete the process.

not sure if this is something to assign to wu or gloria?

**Comment by webdev604** [comment]
hi @gloriajw oneTimePurchase mutation is called with card info on front. that's all. can you check what happens on backend?

**Comment by gloriajw** [comment]
We call Stripe. @helenvarley please log into Stripe and make sure your account is still active.

**Comment by helenvarley** [comment]
yes the account is active - we received a payment via stripe just last week, using the form that is linked to from the website sidebar widget (that goes to this page: https://donate.stripe.com/bIY6oX7SofCY5mE288)

**Comment by helenvarley** [comment]
hi, it would be good to look at this again & get it working, as we're going to need donations ...

i've closed the related issue #1009 - adding a toggle to display or hide the donations strip, which has been done, but that issue also included that the text is not really appropriate - it would be better to say simply "Donate to UpStage" rather than "Pay what you can" as this is not a ticket payment, it's simply a donation. 

we need to relook at ticketing as well but that is a separate issue, #93

**Comment by helenvarley** [comment]
testing again today. when i try a test donation (using test car numbers from Stripe), i click the donate button and nothing happens.

![Image](https://github.com/user-attachments/assets/a3fcfea7-7940-491d-ab65-943b711ed9cd)

**Comment by helenvarley** [comment]
this has been tested with the test cards, & according to those tests it's working. there still needs to be the receipt procedure, but i will close this because it should be working now 😃
